cromwell_tools/_cromwell_auth.py

```diff
@@ -0,0 +1,91 @@
+import json
+import requests
+import requests.auth
+from oauth2client.service_account import ServiceAccountCredentials
+from ._cromwell_api import CromwellAPI
+
+
+class AuthenticationError(Exception):
+    pass
+
+
+class CromwellAuth:
+
+    def __init__(self, url, header, auth):
+        """Authentication Helper for a Cromwell Server
+
+        :param dict header: authorization header
```

Comment:
Can you update this docstring to include info about the `url` and `auth` parameters?

Reply:
Yes! very generally I have not addressed either documentation or testing in this PR. Feel free to make these comments, but know that I will be updating each of the methods and properties with appropriate documentation before any PR gets merged. 

How I document depends somewhat on the conclusion of issue #26.